From 684426ba6a623ee2473c852152dee013652ca49c Mon Sep 17 00:00:00 2001
From: Nils Philippsen <nils@redhat.com>
Date: Wed, 6 Mar 2013 15:28:29 +0100
Subject: [PATCH] patch: mime-types

Squashed commit of the following:

commit 7b32db4ac0653fa375258287843b2dabe57dadfc
Author: Kevin Cozens <kcozens@svn.gnome.org>
Date:   Sun Dec 30 10:43:50 2012 -0500

    Fixed MIME_TYPES. Postscript and PDF are compiled conditionally. Fixes #690882.
    (cherry picked from commit d17622dce6228bebd646ed1eb3063fc952905cfd)
---
 configure.ac | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 2e66736..c32eff3 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1024,7 +1024,7 @@ AC_SUBST(GIMP_COMMAND)
 
 # The list of MIME types that are supported by plug-ins that are compiled
 # unconditionally:
-MIME_TYPES="application/postscript;application/pdf;image/bmp;image/g3fax;image/gif;image/x-fits;image/x-pcx;image/x-portable-anymap;image/x-portable-bitmap;image/x-portable-graymap;image/x-portable-pixmap;image/x-psd;image/x-sgi;image/x-tga;image/x-xbitmap;image/x-xwindowdump;image/x-xcf;image/x-compressed-xcf;image/x-gimp-gbr;image/x-gimp-pat;image/x-gimp-gih"
+MIME_TYPES="image/bmp;image/g3fax;image/gif;image/x-fits;image/x-pcx;image/x-portable-anymap;image/x-portable-bitmap;image/x-portable-graymap;image/x-portable-pixmap;image/x-psd;image/x-sgi;image/x-tga;image/x-xbitmap;image/x-xwindowdump;image/x-xcf;image/x-compressed-xcf;image/x-gimp-gbr;image/x-gimp-pat;image/x-gimp-gih"
 
 
 ###################
@@ -1201,6 +1201,10 @@ if test "x$with_gs" != xno; then
   fi
 fi
 
+if test "x$have_gs" = xyes; then
+  MIME_TYPES="$MIME_TYPES;application/postscript"
+fi
+
 AC_SUBST(FILE_PS)
 AM_CONDITIONAL(HAVE_GS, test "x$have_gs" = xyes)
 AC_SUBST(GS_LIBS)
-- 
1.8.1.4

